frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -184,11 +172,15 @@ const ClusterServiceVersionStatus: React.FC<ClusterServiceVersionStatusProps> =
   obj,
   subscription,
 }) => {
-  const statusString = _.get(obj, 'status.reason', ClusterServiceVersionPhase.CSVPhaseUnknown);
-  const showSuccessIcon = statusString === 'Copied' || statusString === 'InstallSucceeded';
+  const status = _.get(obj, 'status.phase');
```

Comment:
The CLI does not use a default value.  If `status.phase` does not exist, nothing is shown.